02week/style.css

```diff
@@ -42,6 +43,7 @@ p{
     height: 324px;
     background-color: lightgrey;
```

Comment:
For coloring- try googling 'color-picker'. There your can nail down a color and copy the hex#.